rfcs/geometric-primitives.md

```diff
@@ -0,0 +1,349 @@
+# Feature Name: `geometric-primitives`
+
+## Summary
+
+Lightweight geometric primitive types for use across bevy engine crates, and as interoperability types for external libraries.
+
+## Motivation
+
+This would provide a standard way to model primitives across the bevy ecosystem to prevent ecosystem fragmentation amongst plugins.
+
+## User-Facing Explanation
+
+Geometric primitives are lightweight representations of geometry that describe the type of geometry as well as fully defined dimensions. These primitives are *not* meshes, but the underlying precise mathematical definition. For example, a circle is:
+
+```rust
+pub struct Circle2d {
+  origin: Point2d,
+  radius: f32,
+}
+```
+
+Geometric primitives have a defined shape, size, position, and orientation. Position and orientation are **not** defined using bevy's `Transform` components. This is because these are fundamental geometric primitives that must be usable and comparable as-is
```

Comment:
```suggestion
Geometric primitives have a defined shape, size, position, and orientation. Position and orientation are **not** defined using bevy's `Transform` components. This is because these are fundamental geometric primitives that must be usable and comparable as-is.
```

Missing period. I think expanding on why we need "as-is" comparison and use would be valuable here.